content/04.use-cases-b.md

```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
 A CC BY 4.0 license specifies that any code (or other creative products) must be appropriately credited to its original author when distributed, adapted or reused.
 
 
-### Collaborative and asynchronous code editing
+### Collaborative and asynchronous code editing{#code-editing}
 
 <!--*Contributors to this section: Kaitlyn Gaynor, Rob Crystal-Ornelas, Ali, Allison Binley*-->
 
-Researchers can asynchronously communicate and individually or collaboratively work in GitHub.
-By forking, branching, and cloning repositories ([Box 1](#definitions)), researchers can simultaneously work on different issues of the same research project to then later merge changes to the main branch project with pull requests.
-Explicit project organization and increased communication in GitHub Issues, Discussions, or within pull requests can help with project development and with potential merge conflicts due to users simultaneously working on the same sections [@doi:10.1007/s10664-019-09774-x].
-Moreover, version control [@https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html#storing-and-archiving-version-controlled-data] and commit reversal features allow researchers to track and progress without worrying about irreparably modifying someone else's work.
+<!-- I think I accidentally added more jargon, but I feel like this section is really important to get right. I'd be open to leaving all the GitHub jargon out and simply citing a source on this instead.  -->
+
+In academic settings, GitHub can also facilitate interactions between research advisors and advisees, or among collaborators, providing a platform for everyone to share in-progress code, and flag specific challenges or questions for their supervisors, mentors, or collaborators ([Table 2](#tbl:roles)).
+Periodic code review [@doi:10.1145/3341525.3387370] can also help identify errors early in the scientific process, and inform further training and mentorship to fill gaps in skills.
+This is facilitated by a group of core features of git and GitHub that allow contributors to work simultaneously on the same project without interferring with eachother's work.
+"Branching" and "forking" (see [Box 1](#definitions)) allow users to work independently on a linked copy of a repository and then later merge their changes into the main project. 
+To signal that changes are ready to be merged, one makes a "pull request" (see [Box 1](#definitions)).
```

Comment:
I like this section and think it highlights one of the major benefits of using GitHub in scientific research. One thing that may be helpful to touch on, however, is the frequency of when you want to be creating pull requests (or in other words what is the link between branching/forking and pull requests?). Something like "After a team member reaches a new milestone, or addresses issues in code that is ready to be merged one can make a "pull request"... Not sure if it's worth it but we could also add a sentence that discusses something such as "GitHub facilitates team members to frequently branch, commit, and merge code so that large projects avoid spending large amounts of time making software environments, functions, and other portions of their code compatible with other team members when products need to be finished for publication" This would also help to address some of the reviewers comments about how investing in time on the front end can avoid long term issues when code needs to be synchronized for final products. Let me know what you think

Reply:
I don't know, we mention many frequent commits in previous sentence. And I think it's fine to just talk generally within this paragraph about the different git commands like branch, fork, pull without going into "best practices". If we want to include those could put something in the definition table.

Reply:
I see what you're saying @helenweierbach.  I think our readers would probably find some general guidelines useful (like PR == new milestone), but I'm reluctant to make specific recommendations since I think this stuff depends a lot on the group culture.  How about this?
```suggestion
When a set of related changes contributing some new feature or milestone are ready to be evaluated and merged, one makes a "pull request" (see [Box 1](#definitions)).
```